### PR TITLE
Fixed #121: Added and removed icons

### LINKED ISSUE #121: Links
More links need to be added to the footer and not working working should be removed
I am working on it

### PR DESCRIPTION
**Pull request by samyakbvs**
Added the Dribbble link and icon + Removed the bugged GitHub links in the footer as well as the Header

Test link - https://samyakbvs.github.io/samyakjain-jainsamyak-2012.fossasia.org/

Closes #121

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -30,15 +30,15 @@
                     </div>
                     <div class="row ui-sortable">
                         <div class="col-sm-12 text-center">
-                            <ul class="social-profiles">
+                            <ul class="social-profiles"> 
+                                <li><a target="_self" href="https://www.fossasia.net"><i class="icon social_dribbble"></i></a></li>
```
Comment (vaibhavsingh97) [inline _includes/footer.html]: this is not a dribble link

Comment (vaibhavsingh97) [inline _includes/footer.html]: This is not a dribble link

**Comment by samyakbvs** [comment]
But its used on Gci17.fossasia.org as well?

**Comment by samyakbvs** [comment]
@vaibhavsingh97 so should I remove it from there too?

File: _includes/footer.html [hunk 2]
```diff
@@ -30,15 +30,15 @@
                     </div>
                     <div class="row ui-sortable">
                         <div class="col-sm-12 text-center">
-                            <ul class="social-profiles">
+                            <ul class="social-profiles"> 
+                                <li><a href="https://in.pinterest.com/pin/390476230184177529/"><i class=" ti-pinterest" aria-hidden="true"></i></a></li>
```
Comment (vaibhavsingh97) [inline _includes/footer.html]: Remove it as I can find any pinterst link of FOSSASIA. It's a post link

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by vaibhavsingh97** [comment]
> But its used on Gci17.fossasia.org as well?

Please remove from there. Feel free to raise issue and fix it

**Comment by samyakbvs** [comment]
Revised test link - https://samyakbvs.github.io/dumdaayadaadum-jainsamyak-2012.fossasia.org/

**Comment by arjunpandey2003** [comment]
@samyakbvs you have two target attrs in the google plus icon
screen shot 2017-12-31 at 3 11 30 pm

**Comment by samyakbvs** [comment]
@arjunpandey2003 I think its supposed to be like that, bcs I didn't add it

**Comment by arjunpandey2003** [comment]
@samyakbvs ohk

**Comment by harshit98** [comment]
@samyakbvs There should be only 1 target value. Remove the _self value of target.

**Review by harshit98 — APPROVED** [review]

**Pull request merged**